src/boot/z_locale.c

```diff
@@ -6,10 +6,23 @@ s32 gCurrentRegion = 0;
 LocaleCartInfo sCartInfo;
```

Comment:
the only data changes are that `sCartInfo` is gone in N64 versions. Does it work as in-function static? That would combine the `#if`s and have more "parallel structure"